docs/antrea-network-policy.md

```diff
@@ -434,26 +434,26 @@ kind: ClusterNetworkPolicy
 metadata:
   name: acnp-with-igmp-drop
 spec:
-    priority: 5
-    tier: securityops
-    appliedTo:
-      - podSelector:
-          matchLabels:
-            app: mcjoin6
-    ingress:
-      - action: Drop
-        protocols:
-        - igmp:
-            igmpType: 0x11
-            groupAddress: 224.0.0.1
-        name: dropIGMPQuery
-    egress:
-      - action: Drop
-        protocols:
-        - igmp:
-            igmpType: 0x16
-            groupAddress: 225.1.2.3
-    name: dropIGMPReport
+  priority: 5
```

Comment:
If you are adjusting the indent from 4 spaces to 2 spaces, then the following yaml in section `ACNP for multicast egress traffic` also need to be refined.

Reply:
updated